test/cli/cli.go

```diff
@@ -16,23 +16,24 @@ package cli
 
 import (
 	"fmt"
-	"github.com/google/uuid"
-	"github.com/onosproject/onos-config/test/console"
-	"github.com/onosproject/onos-config/test/runner"
-	"github.com/spf13/cobra"
 	"io"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 	"text/tabwriter"
 	"time"
+
+	"github.com/google/uuid"
+	"github.com/onosproject/onos-config/test/console"
+	"github.com/onosproject/onos-config/test/runner"
+	"github.com/spf13/cobra"
 	corev1 "k8s.io/api/core/v1"
 
 )
 
-// GetCommand returns a Cobra command for tests in the given test registry
-func GetCommand(registry *runner.TestRegistry) *cobra.Command {
+// GetOnitCommand returns a Cobra command for tests in the given test registry
```

Comment:
maybe I'm missing the point, but what is the difference between onit and onit-k8s ?

Reply:
I talked this over with @kuujo -- Tests are presumably written for integration and with the assumption devices are also running in the same cluster, so running locally doesn't really make sense. They have to be run in k8s, so we should really probably just add a separate command to run the actual test functions. We split up `onit` is user facing and `onit-k8s` should only be called by onit to run in k8s.